index.bs

```diff
@@ -656,10 +662,14 @@ To <dfn>process the lock request queue</dfn> |queue|:
         1. Let |mode| be |request|'s [=lock request/mode=].
         1. Let |callback| be |request|'s [=lock request/callback=].
         1. Let |p| be |request|'s [=lock request/promise=].
+        1. Let |signal| be |request|'s [=lock request/signal=].
         1. Let |waiting| be [=a new promise=].
         1. Let |lock| be a new [=lock-concept|lock=] with [=lock-concept/agent=] |agent|, [=lock-concept/clientId=] |clientId|, [=lock-concept/origin=] |origin|, [=lock-concept/mode=] |mode|, [=lock-concept/name=] |name|, [=lock-concept/released promise=] |p|, and [=lock-concept/waiting promise=] |waiting|.
         1. [=set/Append=] |lock| to |origin|'s [=held lock set=].
         1. [=Enqueue the following steps=] on |callback|'s [=relevant settings object=]'s [=responsible event loop=]:
+            1. If |signal| is present, then run these steps:
+                1. If |signal|'s [=AbortSignal/aborted flag=] is set, then return.
```

Comment:
Does this leave the lock held, or will it be immediately dropped?

Reply:
Oops, sorry, I hope the new commit fix this.